### PR TITLE
dtr bar display

### PR DESCRIPTION
**Pull request by xanunderscore**
With icon: 
image

without: 
image

umbra version: 
image

popup: 
image

the popup is kind of small but it is the exact same scale as the regular autorotation window - i wonder if it would be a good idea to make it bigger...

### CODE REVIEW

**Comment by adambennett55** [comment]
Looks nice I'll remove the strbat from my pull In lieu of yours

File: BossMod/Framework/Plugin.cs [hunk 1]
```diff
@@ -81,6 +82,9 @@ public unsafe Plugin(IDalamudPluginInterface dalamud, ICommandManager commandMan
         _ai = new(_rotation, _amex);
         _broadcast = new();
         _ipc = new(_rotation, _amex);
+        _dtr = new(_rotation);
+
+        Service.Framework.Update += OnUpdate;
```
Comment (awgil) [inline BossMod/Framework/Plugin.cs:87]: Why do you need it to happen on framework update, rather than on usual plugin update (which happens during draw)?

Comment (xanunderscore) [reply BossMod/Framework/Plugin.cs:87]: oh no idea I copied this from vnavmesh, i can move it

Comment (awgil) [reply BossMod/Framework/Plugin.cs:87]: When you update it, pls merge to master, i've recently merged other PR which added some dtr stuff, I think it's duplicated now.

**Comment by xanunderscore** [comment]
Updated, I moved herc's DTR bar into this class

**Pull request merged**